assignments/a6.md

```diff
@@ -51,7 +56,33 @@ It is suggested to use a folder and multiple files, just like this CSC491 repo,
+| Design | The application has a consistent look and feel. You didn't rely on default styles (see below). The flows are not incoherent. This will be a highly subjective area, so feel free to ask for advice before submission (please don't ask the day before, or day of, submission). | 10.0 |
+| | **Total:** | 100.0 | 
+
+### Notes
+
+* If you have `n` parts of the app (e.g. frontend, backend, data pipeline, etc.), regardless of if they are in different repos or just different folders, each section will be worth `worth/n` for each part of the app.
+* If your code is not under the DCSIL organization, it will be considered as non-existent. You will receive a 0 for that part of your codebase.
+* If your application cannot be easily run and evaluated (hosted on an accessible URL, mobile app installable, desktop app installable, etc.), it will be considered incomplete and receive a 0 on all parts that require us to run the software.
+* If your application _is not_ uploading test coverage to codecov.io, it will be considered to have a coverage of 0%
+* The short write up should be adequately long to tell us what to expect and highlight what you think is important. It should be short and concise enough that we're not bored reading it or lose understanding. (Tell a good story). Feel free to include images (and flow charts!!) if you feel they will help.
+* The short write up should be a markdown file in the repository or the body of the release. If it is a markdown file **include it in the release body**
+* Failing to adhere to any of the requirements, even if it's just "record the URL in a certain spot" will result in reasonable penalities. These kind of tedious tasks are in place to help the teaching staff mark, but they're also going to show up sometimes in companies (so get used to them haha). We appreciate your help in making these assignments more streamlined to grade.
+* Dynamic Content: It's ok if you seed your database with static content, it's not ok if you hard code an HTML table or something. In the former, you would substitute real data and it would "just work", in the latter nothing has been done besides making a UI.
+* Default Styles: The design doesn't need to be state of the art, super modern, and feel like it's been designed by Jony Ive, but it does need to look decent.
+   - Use a consistent colour scheme (recommend looking at a site like [https://paletton.com](https://paletton.com), docs on this in the DCSIL app).
+   - Use a framework/template like [one of these](https://dev.to/sm0ke/react-dashboards-open-source-apps-1c7j), or design something that is coherent.
+   - If you use something like Material UI or Bootstrap, you're using components consistently and aren't relying on the default colours
+   - We want to give our users a nice experience, not confuse them.
```

Comment:
This is good! I think it could say a bit more about how they shouldn't confuse users (like I mentioned above).